wrappers/python/pyrs_device.cpp

```diff
@@ -52,6 +54,17 @@ void init_device(py::module &m) {
                 ss << "  on USB" << self.get_info( RS2_CAMERA_INFO_USB_TYPE_DESCRIPTOR );
             ss << ")>";
             return ss.str();
+        })
+        .def("is_metadata_enabled", [](const rs2::device& self) -> bool{
+            auto depth_sens = self.first< rs2::depth_sensor >();
+
+            if (!depth_sens.supports(RS2_CAMERA_INFO_PHYSICAL_PORT))
+            {
+                throw librealsense::invalid_value_exception("Device does not support checking metadata with thi API");
+            }
+            std::string id = depth_sens.get_info(RS2_CAMERA_INFO_PHYSICAL_PORT);
+
+            return rs2::metadata_helper::instance().is_enabled(id);
```

Comment:
This will be compiled in Linux, too -- but metadata-helper.cpp is linked with only in Windows.
I think we should always link with it.